RF24Network.cpp

```diff
@@ -447,59 +447,59 @@ uint8_t RF24Network::enqueue(RF24NetworkHeader* header)
             IF_SERIAL_DEBUG_FRAGMENTATION_L2(for (int i = 0; i < frag_queue.message_size; i++) { printf_P(PSTR("%02x"), frag_queue.message_buffer[i]); });
             return true;
         }
-        else // NETWORK_MORE_FRAGMENTS
-            if (header->type == NETWORK_LAST_FRAGMENT || header->type == NETWORK_MORE_FRAGMENTS) {
+        // else if not first fragment
+        else if (header->type == NETWORK_LAST_FRAGMENT || header->type == NETWORK_MORE_FRAGMENTS) {
```

Comment:
clang-fomat had a problem here. I must admit it was a little hard to read myself, so I moved around the comment and let clang-format fix the indentation.